app/views/contexts/thank_you_context.py

```diff
@@ -39,7 +39,10 @@ def build_thank_you_context(
     else:
         submission_text = lazy_gettext("Your answers have been submitted.")
     context_metadata = build_submission_metadata_context(
-        survey_type, submitted_at, metadata.tx_id
+        survey_type,
+        submitted_at,
+        # type ignore: tx_id always exists at this point
+        metadata.tx_id,  # type: ignore
```

Comment:
If metadata is a `dict` then calling `metadata.tx_id` is going to error, might be one for @berroar to confirm as I think the change was something you both talked about? would it be better to change `metadata = get_metadata(current_user) or {}` into defaulting to another proxy or just type ignoring it?

Reply:
This method should expect a metadata proxy. It is not valid to call it without one. Therefore, we should change `metadata = get_metadata(current_user) or {}` to 
`metadata: MetadataProxy= get_metadata(current_user)  # type ignore`

Reply:
@mgdon650 - this one is still outstanding I think you can actually just delete the type ignore? now you've fixed the other place it shouldn't be required